app/less/components.less

```diff
@@ -92,3 +92,18 @@ pick-a-date, pick-a-time {
     }
   }
 }
+
+/****** min and max date styles *****/
+.date-alignment{
+    padding-top: 6px;
+}
+.valid-range-msg{
+ min-width:97px;
```

Comment:
Is this needed? I guess you were trying to make the text wrap better. Is this still an issue with the shorter message?